pkg/cmd/create/install.go

```diff
@@ -446,8 +440,14 @@ func (options *InstallOptions) CheckFlags() error {
 		}
 	}
 
+	if options.BatchMode && !flags.NextGeneration && !flags.Tekton && !flags.Prow {
+		flags.Tekton = true
+		flags.Prow = true
+		flags.Kaniko = true
+		options.InitOptions.Flags.NoTiller = true
+	}
```

Comment:
Is this relevant to removing static jenkins master from `jx`?

Reply:
By this I mean this whole `if` block

Reply:
Yup.

Reply:
Should we not remove the registration of these flags from Cobra. I mean these lines:

```golang
	cmd.Flags().BoolVarP(&flags.Prow, prowFlagName, "", false, "Enable Prow to implement Serverless Jenkins and support ChatOps on Pull Requests")
	cmd.Flags().BoolVarP(&flags.Tekton, tektonFlagName, "", false, "Enables the Tekton pipeline engine (which used to be called knative build pipeline) along with Prow to provide Serverless Jenkins. Otherwise we default to use Knative Build if you enable Prow")
```

We could still have the flags per se if it is not so easy to get rid of them initially, but what is the point of having these CLI flags if in the end we are setting them to true anyways?